src/main/java/online/talkandtravel/controller/http/ChatController.java

```diff
@@ -110,36 +111,38 @@ public ResponseEntity<Void> updateLastReadMessage(
   }
 
   /** finds messages that was before specified last read message (including last read message) */
-  @GetMapping("/{chatId}/messages/read")
+  @GetMapping("/chats/{chatId}/messages/read")
   public Page<MessageDtoBasic> getReadMessages(
-      @PathVariable Long chatId, @PageableDefault(sort = "creationDate") Pageable pageable) {
+      @PathVariable Long chatId,
+      @PageableDefault(sort = "creationDate") Pageable pageable) {
     return chatService.findReadMessages(chatId, pageable);
   }
 
   /** finds messages that was sent after specified last read message */
-  @GetMapping("/{chatId}/messages/unread")
+  @GetMapping("/chats/{chatId}/messages/unread")
   public Page<MessageDtoBasic> getUnreadMessages(
-      @PathVariable Long chatId, @PageableDefault(sort = "creationDate") Pageable pageable) {
+      @PathVariable Long chatId,
+      @PageableDefault(sort = "creationDate") Pageable pageable) {
     return chatService.findUnreadMessages(chatId, pageable);
   }
 
-  @GetMapping("/user/{userId}")
-  public List<PrivateChatInfoDto> findUserChats(@PathVariable Long userId) {
-    return chatService.findUserChats(userId);
+  @GetMapping({"/chats/user/{}", "/v2/user/chats"})
+  public List<PrivateChatInfoDto> findUserChats() {
```

Comment:
Now there are `PrivateChatInfoDto` and `ChatInfoDto` and they are fully similar. Here should be return type `ChatIndoDto`